application/btsieve/src/routes.rs

```diff
@@ -56,11 +62,11 @@ pub fn customize_error(rejection: Rejection) -> Result<impl Reply, Rejection> {
         let code = err
             .http_api_problem
             .status
-            .unwrap_or(HttpStatusCode::InternalServerError);
+            .unwrap_or(HttpStatusCode::INTERNAL_SERVER_ERROR);
         let json = warp::reply::json(&err.http_api_problem);
         return Ok(warp::reply::with_status(
             json,
-            StatusCode::from_u16(code.to_u16()).unwrap(),
+            StatusCode::from_u16(code.as_u16()).unwrap(),
```

Comment:
:x: This should be the same `StatusCode` now, no need for a conversion :)

Reply:
good catch